view/view_manager.py

```
@@ -0,0 +1,38 @@
+
+'''
+MainWindow class contains main elements of GUI:
+    - MenuFrame: Frames with buttons to switch between views.
+    - ViewFrame: Frames to display content/data.
+    - Window: The main window container for the application.
+'''
+
+
+class ViewManager:
+
+    def __init__(self):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)